tests/test_quickfiles.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from osf.models import QuickFiles, BaseFileNode
+from osf_tests.factories import AuthUserFactory
+
+from tests.base import test_app
+from webtest_plus import TestApp
+from addons.osfstorage.tests.utils import make_payload
+
+from framework.auth import signing
+
+
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
+
+
+@pytest.fixture()
+def quickfiles(user):
+    return QuickFiles.objects.get(creator=user)
+
+
+@pytest.fixture()
+def flask_app():
+    return TestApp(test_app)
+
+
+@pytest.fixture()
+def post_to_quickfiles(quickfiles, user, flask_app, **kwargs):
+    def func(name, *args, **kwargs):
+        osfstorage = quickfiles.get_addon('osfstorage')
+        root = osfstorage.get_root()
+        url = '/api/v1/project/{}/osfstorage/{}/children/'.format(quickfiles._id, root._id)
```

Comment:
For clarification on the review, I posted here to v1 to mimic the structure of [other osfstorage upload tests](https://github.com/CenterForOpenScience/osf.io/blob/develop/addons/osfstorage/tests/test_views.py#L146) without re-doing or re-implementing the structure of the Hook test cases. Simply posting to the v1 route seemed more straightforward